src/dftbp/dftbplus/inputdata.F90

```diff
@@ -108,6 +108,21 @@ module dftbp_dftbplus_inputdata
     !> Separation parameter
     real(dp) :: omega
 
+    !> CAM alpha parameter
+    real(dp) :: camAlpha
+
+    !> CAM beta parameter
+    real(dp) :: camBeta
+
+    !> True, for global hybrids
+    logical :: tHyb
+
+    !> True, for long-range corrected functionals
+    logical :: tLc
+
+    !> True, for general CAM range-separation
+    logical :: tCam
```

Comment:
Can we make this an enumerator? Three independent logicals are hard to track, instead we could just use an enumerator (maybe `exchange_kernel`) with the values `none`, `hybrid`, `longRange`, and `cam` (or similar).